moon/config/moon-round1.json

```diff
@@ -48,6 +48,7 @@
               ["/scout_1/camera/left/image_raw/compressed", "sensor_msgs/CompressedImage", "left_image"],
               ["/scout_1/camera/right/image_raw/compressed", "sensor_msgs/CompressedImage", "right_image"],
               ["/scout_1/imu", "sensor_msgs/Imu", "orientation"],
+              ["/scout_1/openvslam/pose", "geometry_msgs/PoseStamped", "vslam_pose"],
```

Comment:
and vslam pose is published separately anyway ... so I would undo the pose2d-> odo_pose rename

Reply:
Again, it is renamed to indicate that it is an estimate coming from odometry, similarly vslam_pose indicates that this is an estimate coming in from VSLAM. These inputs will be fused and only from that fused value a stream called 'pose' will be produced.